pyguide.md

```diff
@@ -1982,16 +1982,40 @@ aptly described using a one-line docstring.
 
 <a id="doc-function-args"></a>
 [*Args:*](#doc-function-args)
-:   List each parameter by name. A description should follow the name, and be
-    separated by a colon and a space. If the description is too long to fit on a
-    single 80-character line, use a hanging indent of 2 or 4 spaces (be
-    consistent with the rest of the file).
+:   List each parameter by name. A description should follow the name on the
+    same line and be separated by a colon and a space. A description can also
+    follow on the next line. If the description is too long to fit on a single
+    80-character line, use a hanging indent of 2 or 4 spaces (be consistent with
+    the rest of the file).
 
     The description should include required type(s) if the code does not contain
     a corresponding type annotation. If a function accepts `*foo` (variable
     length argument lists) and/or `**bar` (arbitrary keyword arguments), they
     should be listed as `*foo` and `**bar`.
 
+    Both examples below are valid descriptions.
+
+```python
+def print_data(verbose):
+    """Prints information about the user's environment.
+
+    Args:
```

Comment:
Can we also clarify what to do in case of a mix of both the styles? And add an example for that as well?

Reply:
If I recall, there'd been some discussion [here](https://github.com/PyCQA/pydocstyle/issues/449) in favor of allowing mixed styles across a code base, but not within a single docstring. That may be over prescriptive though and should probably be decided by someone at google. cc: @gpshead

Reply:
in general i favor not being over prescriptive about that.  the main consistency thing is the indentation level (which is already mentioned).  but trying to enumerate a matrix of possible argument or exception and their descriptions style mixes for things would explode meaninglessly and become an enemy of documentation. :)

Reply:
As long as "hanging indent" style is allowed by the standard, I'm happy for maintainers of tools to decide whether or not to enforce consistency. Avoiding being over prescriptive and having the option to pick the right look for the right doc would be a great thing.